### ISSUE TITLE
Bot stopped working.

### ISSUE THREAD

**Issue by bolex17** [nwithan8/Tauticord#14]
Hi team,

The bot was working all yesterday but now won't work?
I've tried re-installing, creating another bot etc and still nothing.

Have tried running via docker and natively.

Below is the error message I get

```
`WARNING:PyNaCl is not installed, voice will NOT be supported

Ignoring exception in on_ready

Traceback (most recent call last):

  File "/usr/local/lib/python3.9/site-packages/discord/client.py", line 343, in _run_event

    await coro(*args, **kwargs)

  File "/app/modules/discord_connector.py", line 104, in on_ready

    await start_bot(discord_connector=self, analytics=self.analytics)

  File "/app/modules/discord_connector.py", line 24, in start_bot

    message = await discord_connector.edit_message(previous_message=message)

  File "/app/modules/discord_connector.py", line 119, in edit_message

    new_message, count, activity = self.tautulli.refresh_data()

  File "/app/modules/tautulli_connector.py", line 216, in refresh_data

    overview_message = activity.message

  File "/app/modules/tautulli_connector.py", line 49, in message

    if self.total_bandwidth:

  File "/app/modules/tautulli_connector.py", line 30, in total_bandwidth

    return utils.human_bitrate(float(json_bandwidth))

  File "/app/modules/utils.py", line 35, in human_bitrate

    return f'{int(B / KB):.{d}} kbps'

ValueError: Precision not allowed in integer format specifier
```

**Comment by bolex17** [comment]
It seems to have fixed itself

**Comment by nwithan8** [comment]
Fixed in 329309cfc79fb3664d2d3d18f18b1f487cf5b216
